c/tskit/tables.c

```diff
@@ -10087,6 +10179,41 @@ tsk_table_collection_copy(const tsk_table_collection_t *self,
         goto out;
     }
 
+    tsk_reference_sequence_free(dest->reference_sequence);
```

Comment:
I see, this is where free testing for NULL is handy. Still, easy to do 
```c
if (dest->reference_sequence != NULL) {
    tsk_reference_sequence_free(dest->reference_sequence);
}
```

Reply:
Done